src/main/java/com/theocean/fundering/domain/celebrity/repository/CelebRepositoryImpl.java

```diff
@@ -0,0 +1,89 @@
+package com.theocean.fundering.domain.celebrity.repository;
+
+import com.querydsl.core.types.Projections;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import com.theocean.fundering.domain.celebrity.dto.CelebFundingResponseDTO;
+import com.theocean.fundering.domain.celebrity.dto.CelebListResponseDTO;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
+import org.springframework.data.domain.SliceImpl;
+
+import java.util.List;
+
+import static com.theocean.fundering.domain.celebrity.domain.QCelebrity.*;
+import static com.theocean.fundering.domain.post.domain.QPost.post;
+
+@RequiredArgsConstructor
+@Slf4j
+public class CelebRepositoryImpl implements CelebRepositoryCustom {
+    private final JPAQueryFactory queryFactory;
+
+    @Override
+    public Slice<CelebFundingResponseDTO> findAllPosting(Long celebId, Long postId, Pageable pageable) {
+        List<CelebFundingResponseDTO> contents = queryFactory
+                .select(Projections.constructor(CelebFundingResponseDTO.class,
+                        post.postId,
+                        post.writer.userId,
+                        post.writer.nickname,
+                        post.celebrity.celebId,
+                        post.celebrity.celebName,
+                        post.title,
+                        post.content,
+                        post.participants,
+                        post.targetPrice))
+                .from(post)
+                .where(eqPostCelebId(celebId), ltPostId(postId))
+                .orderBy(post.postId.desc())
+                .limit(pageable.getPageSize())
+                .fetch();
+        boolean hasNext = false;
+        if(contents.size() > pageable.getPageSize()){
+            hasNext = true;
+        }
+        return new SliceImpl<>(contents, pageable, hasNext);
+    }
+
+    @Override
+    public Slice<CelebListResponseDTO> findAllCeleb(Long celebId, Pageable pageable) {
+        List<CelebListResponseDTO> contents = queryFactory
+                .select(Projections.constructor(CelebListResponseDTO.class,
+                        celebrity.celebId,
+                        celebrity.celebName,
+                        celebrity.celebGender,
+                        celebrity.celebType,
+                        celebrity.celebGroup,
+                        celebrity.profileImage))
+                .from(celebrity)
+                .where(ltCelebId(celebId))
+                .orderBy(celebrity.celebId.desc())
+                .limit(pageable.getPageSize())
+                .fetch();
+        boolean hasNext = false;
+        if(contents.size() > pageable.getPageSize()){
+            hasNext = true;
+        }
```

Comment:
code inspection을 잘 설정하면 이렇게 IntelliJ 가 추천을 잘 해줍니다.
<img width="1198" alt="Screenshot 2023-10-19 at 1 32 45 AM" src="https://github.com/Step3-kakao-tech-campus/Team13_BE/assets/5006605/c0fbe8c3-2075-4e9b-bead-3feb999ebaec">

```suggestion
        final boolean hasNext = contents.size() > pageable.getPageSize();
```